live/global.ini

```diff
@@ -7362,7 +7362,7 @@ DynamicEvent_FleetWeek2022_Generic_IslandBossMarker=Neutralisieren
 DynamicEvent_FleetWeek2022_Generic_MopUpHUD=Feinde der Plattform eliminieren:
 DynamicEvent_FleetWeek2022_Generic_RegainAntiAir_long=Neutralisiere die Luftabwehr der Plattform, indem Sie sie deaktivieren oder die Kontrolle darüber wiedererlangen.
 DynamicEvent_FleetWeek2022_Generic_RegainAntiAir_short=Luftabwehr der Plattform neutralisieren
-DynamicEvent_FleetWeek2022_Generic_SearchCargo_Barge_long=Durchsuchen Sie die Frachtcontainer von Pressley Party Supply auf dem Lastkahn nach dem IFFI mit dem Code ~mission(BargeCode).
+DynamicEvent_FleetWeek2022_Generic_SearchCargo_Barge_long=Durchsuchen Sie die Frachtcontainer von Pressley's Party Supply auf dem Frachtschiff nach dem IFFI mit dem Code ~mission(BargeCode).
```

Comment:
Änderung: `"Wir haben jetzt Zugriff zum Inverter auf dem Frachtschiff,"`

`auf den... auf dem` klingt blöd in einem Satz so kurz hintereinander. Ein Inverter ist das ein Wechselrichter?